crates/oci-distribution/src/lib.rs

```diff
@@ -0,0 +1,174 @@
+#[macro_use]
+extern crate serde;
```

Comment:
personal nit: I prefer referring to macros using fully qualified paths instead of using `#[macro_use]`

Reply:
I tried this out, and I agree. It is actually more readable all around.